dts/bindings/i2c/nxp,imx-lpi2c.yaml

```diff
@@ -18,3 +18,15 @@ properties:
       required: false
       type: int
       description: Bus idle timeout in nanoseconds
+
```

Comment:
Wonder about putting these in a `i2c/gpio-bus-recovery.yaml` so they can be included and common across a number of i2c controllers.

Reply:
It would be nice to standardize across I2C controller bindings, agreed. The Linux kernel devicetree bindings have these on the top-level: https://www.kernel.org/doc/Documentation/devicetree/bindings/i2c/i2c.txt

How about adding them directly to `i2c-controller.yaml`? That would allow re-using them for `gpio-i2c.yaml` as well. We can exclude them from I2C controller bindings not supporting them, but having them in the top-level forces developers to consider them? I am not sure which would be the best option.

Reply:
agreed, would be nice to make them to `i2c-controller.yaml`

Reply:
Should we use `property-blocklist` for I2C controllers not using them (yet) if we pull them into `i2c-controller.yaml`?